tensorboard/plugins/beholder/__init__.py

```diff
@@ -12,5 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from tensorboard.plugins.beholder.beholder import Beholder
-from tensorboard.plugins.beholder.beholder import BeholderHook
+try:
```

Comment:
Let's only put `import tensorflow` inside the try-except, and then if that succeeds, do the beholder imports outside a try-except.  That ensures that they will be missing only when the import error is what we expect (an error finding tensorflow) rather than if they have other import errors, which would be an actual error we wouldn't want to suppress.

```python
# Only import Beholder API when tensorflow is available.
try:
  import tensorflow
except ImportError:
  pass
else:
  from tensorboard.plugins.beholder.beholder import Beholder
  from tensorboard.plugins.beholder.beholder import BeholderHook
```